app/src/main/kotlin/co/netguru/baby/monitor/client/feature/noisedetection/NoiseDetector.kt

```diff
@@ -0,0 +1,31 @@
+package co.netguru.baby.monitor.client.feature.noisedetection
+
+import co.netguru.baby.monitor.client.feature.voiceAnalysis.AacRecorder.Companion.SAMPLING_RATE
+import io.reactivex.Single
+import timber.log.Timber
+import javax.inject.Inject
+import kotlin.math.log10
+import kotlin.math.sqrt
+
+class NoiseDetector @Inject constructor() {
+    fun processData(data: ShortArray, size: Int)= Single.fromCallable<Double> {
+        var decibels = 0.0
+        if (size > 0) {
+            var totalSquared = 0.0
+            for (i in 0 until size) {
+                val soundBit = data[i]
+                totalSquared += (soundBit * soundBit).toDouble()
+            }
+
+            val quadraticMeanPressure = sqrt(totalSquared / size)
+            decibels = 20 * log10(quadraticMeanPressure)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/8b49d288c8cda524f66973edfcfe165e7a30812e/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/noisedetection/NoiseDetector.kt#L21"></span>This expression contains a magic number. Consider defining it to a well named constant.</td>
    </tr>
  </tbody>
</table>